logbook-logstash-logback-encoder/src/test/java/org/zalando/logbook/lle/LogbackLogstashLogWriterTest.java

```diff
@@ -0,0 +1,30 @@
+package org.zalando.logbook.lle;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.IOException;
+
+import org.junit.jupiter.api.Test;
+import org.slf4j.Marker;
+import org.zalando.logbook.Correlation;
+import org.zalando.logbook.Precorrelation;
+
+public class LogbackLogstashLogWriterTest {
+
+    @Test
+    public void testDefaultImplementation() {
+        LogbackLogstashLogWriter writer = new LogbackLogstashLogWriter() {
```

Comment:
You can actually create a mock and use `callRealMethod` as the default answer. Saves you ~20 lines.